File: python/Command/SimpleCalc.py

class SimpleCalc:
    def __init__(self) -> None:
        print("CALCULATOR: Creating calculator\n")
        self.sum_ = 0
        self.diff_ = 0
    
    def Sum(self, digits: list[float]) -> float:
        print("CALCULATOR: Calculating sum\n")
        
        if (digits != None):
            self.sum_ = sum(digits)
            return self.sum_
        
    def Diff(self, digits: list[float]) -> float:
        print("CALCULATOR: Calculating difference\n")

        if (digits != None):
            for digit in digits:
                self.diff -= digit
            return self.diff

File: C#/Command/SimpleCalc.cs
using System;
using System.Collections.Generic;
using System.Globalization;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Command
{
    class SimpleCalc
    {
        private double sum;
        private double diff;

        public SimpleCalc()
        {
            Console.WriteLine("CALCULATOR: Creating calculator");
            diff = 0;
            sum = 0;

        }
        public double Sum(List<double> digits)
        {
            Console.WriteLine("CALCULATOR: Calculating sum");

            foreach(double digit in digits)
            {                
                sum += digit;
            }
            return sum;
        }
            
        public double Diff(List<double> digits)
        {
            Console.WriteLine("CALCULATOR: Calculating difference");

            foreach (double digit in digits)
            {
                diff -= digit;
            }
            return diff;
        }
    }
}
